data/tlg0062/tlg008/tlg0062.tlg008.perseus-grc2.xml

```diff
@@ -136,9 +136,10 @@
 </div>
 <div type="textpart" subtype="section" xml:base="urn:cts:greekLit:tlg0062.tlg008.perseus-grc2" n="19">
     <p rend="align(indent)"> τὸν δὲ Κυνικὸν τὸν ἐν ἄρκτου δέρματι φιλοσοφοῦντα οὐχ Ὁνωρᾶτον, ὥσπερ ὠνομάζετο, ἀλλʼ Ἀρκεσίλαον καλεῖν ἠξίου.</p>
-    <p rend="align(indent)">ἐρωτήσαντος δέ τινος, τίς αὐτῷ ὅρος εὐδαιμονίας εἶναι δοκεῖ, μόνον εὐδαίμονα ἔφη τὸν ἐλεύθερον· ἐκείνου δὲ φήσαντος πολλοὺς ἐλευθέρους εἶναι, </p>
+    
 </div>
 <div type="textpart" subtype="section" xml:base="urn:cts:greekLit:tlg0062.tlg008.perseus-grc2" n="20">
+<p rend="align(indent)">ἐρωτήσαντος δέ τινος, τίς αὐτῷ ὅρος εὐδαιμονίας εἶναι δοκεῖ, μόνον εὐδαίμονα ἔφη τὸν ἐλεύθερον· ἐκείνου δὲ φήσαντος πολλοὺς ἐλευθέρους εἶναι, </p>
```

Comment:
I know this is a messy set of texts with lots of capture errors, but if the section breaks differ from print then we usually indicate that this is resp="perseus" and/or add a note. rather than a silent change to the section break.  

This also differs from the other capture:
https://scaife.perseus.org/reader/urn:cts:greekLit:tlg0062.tlg008.1st1K-grc1:20